src/Components/__tests__/ProgressBar.spec.js

```diff
@@ -6,7 +6,6 @@ describe("<ProgressBar />", () => {
   let progressBar;
   const props = {
     currentSectionName: "conv",
-    currentSection: 0,
     totalSections: 7,
   };
   beforeEach(() => {
```

Comment:
On line 15, what does it mean for something to be a progressbar? Maybe a more accurate statement is "it should render a progressbar"? then instead of testing the length, we can test whether the progress bar exists or not.